documentation/web-components/integrating-a-web-component.asciidoc

```diff
@@ -6,7 +6,7 @@ layout: page
 
 = Integrating a Web Component
 
-Web components is a standards based way to create and consume reusable components through custom HTML tags. To learn more about web componnents, see https://www.webcomponents.org/introduction.
+Web Components are a collection of web standards allowing you to create new HTML tags with custom name, reusability and full encapsulation on styles & markup. To understand more about what Web Components are, visit https://vaadin.com/docs/v11/flow/web-components/introduction-to-webcomponents.html[Introduction to Web Components].
```

Comment:
oh, one thing: i think this link should point to https://github.com/vaadin/flow-and-components-documentation/blob/master/documentation/web-components/introduction-to-webcomponents.asciidoc than the link on vaadin.com. 

I have checked the other links used in the docs here, it looks vaadin.com has some treatment to convert those link to Vcom ones.

Reply:
What are Vcom ones? Can you elaborate?

Reply:
Yes, I understand that the link needs to be changed. I just don't understand the reason you said.